src/de/seq.rs

```diff
@@ -74,20 +74,6 @@ where
 {
     /// Deserializer used to deserialize sequence items
     de: &'a mut Deserializer<'de, R>,
-    /// Filter that determines whether a tag is a part of this sequence.
-    ///
-    /// When feature `overlapped-lists` is not activated, iteration will stop
-    /// when found a tag that does not pass this filter.
-    ///
-    /// When feature `overlapped-lists` is activated, all tags, that not pass
-    /// this check, will be skipped.
-    filter: TagFilter<'de>,
```

Comment:
In fact, since this was the only use of `TagFilter` and it is now deleted, I believe all of that is dead code 

https://github.com/tafia/quick-xml/search?q=tagfilter

Reply:
update: no, github search just sucks and doesn't show all uses.  if I checkout the branch locally, `MapValueSeqAccess` is obviously still using it.